package.json

```diff
@@ -39,6 +39,7 @@
     "test": "c8 mocha build/test"
   },
   "dependencies": {
+    "express": "^4.17.1",
```

Comment:
I don't think we need this `express` dependency -- at least I don't see it being used.

Reply:
Removed in https://github.com/googleapis/nodejs-video-transcoder/pull/39/commits/9d65c25a794f74992306219d6c1e3f53ee47c08b